src/features/usage-consolidation-accordion/summary-table/summary-table.js

```diff
@@ -42,18 +47,27 @@ const SummaryTable = ({
   ...rest
 }) => {
   const intl = useIntl();
-  const data = costPerUseData.data[costPerUseType];
+  const summaryMCLRef = useRef(null);
 
-  if (!data) {
-    return null;
-  }
+  const data = costPerUseData.data[costPerUseType];
+  const { isLoaded } = costPerUseData;
 
   const currency = data?.attributes?.parameters?.currency;
 
   const contentData = !noCostPerUseAvailable
     ? rest.contentData || [data?.attributes?.analysis]
     : [];
 
+  useEffect(() => {
+    if (summaryMCLRef.current && isLoaded) {
+      getMCLFirstDataRow(summaryMCLRef.current).focus();
+    }
+  }, [summaryMCLRef, isLoaded]);
+
+  if (!data) {
```

Comment:
Please move this block higher up just below `const summaryMCLRef = useRef(null);`. We don't need to call `useEffect` if we're not going to render anything

Reply:
I also noticed, that this data check is the same as in `usage-consolidation-[entity-type].js`.

We have this part of code in that files: 
``` javascript
  const data = costPerUseData.data[costPerUseTypes.SOME_TYPE];

  if (!data) {
    return null;
  }
```
And the same check is in `SummaryTable`:

```javascript
const data = costPerUseData.data[costPerUseType];

  if (!data) {
    return null;
  }
```
`costPerUseType` is the same as in parent component, so it seems, like in `SummaryTable` data always will be. So I can get rid of this if statement in `SummaryTable`.

What do you think?

Reply:
Ok, let's remove that check in `<SummaryTable`>